### PR TITLE
Add template_fields and constructor parameters to displaying

### LINKED ISSUE #8893: Displaying field templates and constructor parameters in scripts/list-integration.py
Hello,

We have script that allows us to display class names.
https://github.com/apache/airflow/blob/master/scripts/list-integrations.py
It would be nice if it were possible to display template fields and constructor parameters.  This should be available when you run this script with an additional option. The output should be available as CSV or JSON or JSONL. Constructor parameters from BaseOperator or BaseSensor should be ignored.
This tool is needed to review all operators to eliminate missing template fields in existing operators.

Related PR:
https://github.com/apache/airflow/pull/8870
https://github.com/apache/airflow/pull/8840
https://github.com/apache/airflow/pull/7198
https://github.com/apache/airflow/pull/7127
https://github.com/apache/airflow/pull/6656
https://github.com/apache/airflow/pull/5718
https://github.com/apache/airflow/pull/5490
https://github.com/apache/airflow/pull/5248
https://github.com/apache/airflow/pull/5097
https://github.com/apache/airflow/pull/4920
https://github.com/apache/airflow/pull/4691
https://github.com/apache/airflow/pull/3305
https://github.com/apache/airflow/pull/2981
https://github.com/apache/airflow/pull/2549
https://github.com/apache/airflow/pull/2480
https://github.com/apache/airflow/pull/2235
https://github.com/apache/airflow/pull/1808
https://github.com/apache/airflow/pull/996
and more

### PR DESCRIPTION
**Pull request by chenglongyan**
This pr add template_fields and constructor parameters to output of [list-integrations.py](https://github.com/apache/airflow/blob/main/scripts/tools/list-integrations.py) 

### Output format:
####[flag] [class-name];[template_fields];[constructor parameters]
class-name :  full qualified class name
template_fields : comma separated
constructor parameters : comma separated

##### Before add:
image

##### After:
image

closes: #8893

### CODE REVIEW

**Comment by potiuk** [comment]
Can we see a screenshot of the docs generated with it please @chenglongyan ?

**Comment by chenglongyan** [comment]
Screenshot added

> Can we see a screenshot of the docs generated with it please @chenglongyan ?

**Comment by chenglongyan** [comment]
Reopen !!

**Comment by chenglongyan** [comment]
not stale ...

**Comment by potiuk** [comment]
I guess you are going to continue @chenglongyan ?

File: scripts/tools/list-integrations.py [hunk 1]
```diff
@@ -63,7 +63,12 @@ def _find_clazzes(directory, base_class):
         ]
 
         for found_clazz in integration_clazzes:
-            found_classes.add(f"{found_clazz.__module__}.{found_clazz.__name__}")
+            class_name = f"{found_clazz.__module__}.{found_clazz.__name__}"
+            template_fields = ",".join(getattr(found_clazz, 'template_fields', []))
+            init_params = ",".join(
+                getattr(found_clazz.__init__, f"_{found_clazz.__class__.__name__}__param_names", [])
```
Comment (uranusjr) [inline scripts/tools/list-integrations.py]: I think this is wrong? The private variable is always named `_BaseOperatorMeta__param_names`, if I understand correctly.

**Comment by uranusjr** [comment]
We probably should hide this output behind a flag since the argument names can intefere with how the script can be used with `grep`.

**Comment by chenglongyan** [comment]
@potiuk yes, if this tool still works for you.

> I guess you are going to continue @chenglongyan ?

File: scripts/tools/list-integrations.py (hunk 1 above)
Comment (chenglongyan) [reply scripts/tools/list-integrations.py]: thanks @uranusjr , corrected.

**Comment by chenglongyan** [comment]
@uranusjr , added.

> We probably should hide this output behind a flag since the argument names can intefere with how the script can be used with `grep`.

**Pull request closed without merging**